authors/apps/profiles/tests/test_follow.py

```diff
@@ -22,7 +22,10 @@ def setUp(self):
     def save_user(self, username, email, password):
         data = {'username': username,
                 'email': email, 'password': password}
-        return User.objects.create_user(**data)
+        res = User.objects.create_user(**data)
```

Comment:
Hello @Bakley ,
Usually, when we have keyword arguments in a method, it is standard to use `**kwargs` . It is easily identifiable and standard way of using variable length dictionary in python methods.

Reply:
Okay, fixed

Reply:
Oh, I see it is not a function prototype

Reply:
@Ogutu-Brian When changed it fails the test. I researched about using `**kwargs` is mostly used on functions to unpack. Here these tests were unpacking the data dictionary